src/test/java/pages/ShoppingCartPage.java

```diff
@@ -0,0 +1,49 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ShoppingCartPage extends BasePage {
+    private final static By CHECKOUT_BUTTON = By.id("checkout");
+    private final static By CONTINUE_SHOPPING_BUTTON = By.id("continue-shopping");
+    private final static By ITEM_DESCRIPTION_CART = By.className("inventory_item_desc");
+    private final static String ITEM_CONTAINER_CART = "//div[text()='%s']/ancestor::div[@class='cart_item']";
+    private final static By ITEM_PRICE_CART = By.className("inventory_item_price");
+    private final static By REMOVE_BUTTON_CART = By.cssSelector("button[id^=remove]");
+    private final static By SHOPPING_CART_BADGE = By.className("shopping_cart_badge");
+
+    public ShoppingCartPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void CheckoutButton() {
+        driver.findElement(CHECKOUT_BUTTON).click();
+    }
+
+    public void ContinueButton() {
```

Comment:
1. Method names from Capital letter
2. Method names should be a verb, not a noun like: click...Button